### ISSUE TITLE
Inconsistent behavior: Image Description works while Screen Curtain is enabled

### ISSUE THREAD

**Issue by HBM301** [nvaccess/nvda#19045]
### Steps to reproduce

- Open NVDA and enable the AI image description feature
- Enable the screen curtain
- Navigate to any location on the screen (for example desktop screen)
- Press NVDA+Window+Comma to retrieve the image description

### Actual behavior

Even when the Screen Curtain is enabled and the screen is completely black, NVDA still provides an image description
For example, it may announce:
"A close-up of a black and white photo of a building."

### Expected behavior

If the user tries to use the Image Description feature while the Screen Curtain is enabled, NVDA should announce a message such as:
Please disable Screen Curtain before using  AI image descriptions
This helps avoid confusion and makes it clear that the feature requires the screen to be visible in order to work properly

### NVDA type

portable copy

### NVDA version

alpha-52951,121c221f (2026.1.0.52951)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

### Windows version

Windows 11 24H2 (AMD64) build 26100.6725

### Does the issue still occur after restarting your computer?

I have not restarted my computer

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have not run the COM Registration Fixing Tool

**Comment by CyrilleB79** [comment]
True.
On the opposite, OCR reports "Please disable screen curtain before using Windows OCR." when trying to OCR something while the screen curtain is active.

Not only image recognition should report an error message when screen curtain is active. But the User Guide should also be updated to mention such a limitation.

In the user guide we have:
* In section 9.2 Screen curtain
> When the Screen Curtain is active some tasks directly based on what appears on the screen such as performing OCR or taking a screenshot cannot be achieved.
* In section 10.1. Windows OCR
> Windows OCR may be partially or fully incompatible with NVDA vision enhancements or other external visual aids. You will need to disable these aids before proceeding to a recognition.

Does this also apply to image recognition? If yes the User Guide should be updated.

At last, this raises some more question, maybe off-topic here, sorry. From user point of view, Windows OCR and Image recognition are both recognizers:
* why Image recognizer has not been implemented as a recognizer too?
* why image recognizer's description is in a paragraph after NVDA settings while OCR is before? This makes the User Guide structure less understandable, so more difficult to find useful information in it.

Cc @Qchristensen  for UG concerns.

That being said, I acknowledge that developing Image recognition has already been a big challenge, and that's very nice to have it in NVDA. If not possible to do better, it can ship as is and that will be a real plus.

But I really think that when a new feature is integrated in core, we should care as much as possible about UX (e.g. settings / gestures organization) and good documentation, not postpone it.
Thanks.

**Comment by seanbudd** [comment]
@tianzeshi-study - can you please add a `blockAction.Context` for screen curtain to give a warning when activating the gesture for image descriptions

**Comment by tianzeshi-study** [comment]
> [@tianzeshi-study](https://github.com/tianzeshi-study) - can you please add a `blockAction.Context` for screen curtain to give a warning when activating the gesture for image descriptions

Sure, got it.
